src/components/AnonymousChat/PostNotificationChannelItem.tsx

```diff
@@ -12,12 +12,11 @@ import {MessageChannelItemProps} from '../../../types/component/AnonymousChat/Me
 import {calculateTime} from '../../utils/time';
 import {capitalizeFirstText} from '../../utils/string/StringUtils';
 
-const AnonPostNotificationChannelItem: (props: MessageChannelItemProps) => React.ReactElement = ({
+const PostNotificationChannelItem: (props: MessageChannelItemProps) => React.ReactElement = ({
   item,
   onChannelPressed
 }) => {
-  // TODO: Change this into useUserAuthHook
-  const {anonProfileId, signedProfileId} = useProfileHook();
+  const {anonProfileId, signedProfileId} = useUserAuthHook();
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The function name has been changed from `AnonPostNotificationChannelItem` to `PostNotificationChannelItem`, and the `useProfileHook` has been replaced with `useUserAuthHook`. Make sure that this hook provides all the necessary data for the component and that it's properly tested. 

```diff
- const AnonPostNotificationChannelItem: (props: MessageChannelItemProps) => React.ReactElement = ({
-   item,
-   onChannelPressed
- }) => {
-  // TODO: Change this into useUserAuthHook
-  const {anonProfileId, signedProfileId} = useProfileHook();
+ const PostNotificationChannelItem: (props: MessageChannelItemProps) => React.ReactElement = ({
+    item,
+    onChannelPressed
+  }) => {
+   const {anonProfileId, signedProfileId} = useUserAuthHook();
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->